Week3/homework/1-step3.css

```diff
@@ -0,0 +1,10 @@
+body,
```

Comment:
Alejandra, 
Something has definitely not gone right here. 
You have committed (in one commit) all the code for the last 3 weeks.  This makes things much more difficult to sift through and find the work for week 3.
I would suggest that you do a few more git tutorials, as this skill is actually critical when programming in the real world.  
This time it is just you working on the repository, but imagine there are 10 different people working on it, and you suddenly create double commits through big chunks of the repository.  I can promise you (because I've done it) that your co-workers will not be happy with you, and you will be spending hours fixing it. 😄

Reply:
Hi Rohan, thank you for the feedback I definitely need to pay attention to the details. I will fix it and will be more careful in the future.